extensions/opentelemetry-exporter-jaeger/runtime/src/main/java/io/quarkus/opentelemetry/exporter/jaeger/JaegerRecorder.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.opentelemetry.exporter.jaeger;
+
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.spi.CDI;
+
+import io.opentelemetry.exporter.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.quarkus.runtime.annotations.Recorder;
+
+@Recorder
+public class JaegerRecorder {
+    public void installBatchSpanProcessorForJaeger(JaegerExporterConfig.JaegerExporterBuildConfig buildConfig,
+            JaegerExporterConfig.JaegerExporterRuntimeConfig runtimeConfig) {
+
+        // Only create the JaegerGrpcSpanExporter if an endpoint was set in runtime config
+        if (runtimeConfig.endpoint != null && runtimeConfig.endpoint.trim().length() > 0) {
```

Comment:
why do you need to check this again? Isn't the call to the recorder guarded by endpoint being configured already?

Reply:
There's currently no check on the processor.

I could move this endpoint check into the processor instead, and then the recorder method wouldn't be called if it isn't set.

Reply:
Seems more efficient to do it that way.. I could swear I saw you checking that endpoint setting in the deployment module.. but I suppose if it can be changed @ runtime, this way is better? sorry for waffling.. just thought I saw the check elsewhere

Reply:
I will move it to the processor and see if all goes well.

Won't push until tomorrow to wait for additional comments

Reply:
So I tried doing this and had some issues.

Asked a few people and was told that runtime config values can't be relied on in the processor, so the check needs to stay in the recorder

Reply:
y.. that's where I started to land once I realized you were checking runtime config.